src/Creitive/Breadcrumbs/BreadcrumbsServiceProvider.php

```diff
@@ -21,12 +30,21 @@ public function boot()
 	 */
 	public function register()
 	{
+
 		$this->app['breadcrumbs'] = $this->app->share(function($app)
 		{
 			return new Breadcrumbs;
 		});
-
-		$this->app->alias('breadcrumbs', 'Creitive\Breadcrumbs\Breadcrumbs');
```

Comment:
This line shouldn't have been deleted, as it enables users to benefit from Laravel's automatic dependency injection, ie. if they don't want to use the facade in their code, they can just inject the shared instance and call methods on it.